astropy/wcs/wcsapi/sliced_low_level_wcs.py

```diff
@@ -32,9 +36,9 @@ def sanitize_slices(slices, ndim):
             slc = slices[i]
             if isinstance(slc, slice):
                 if slc.step and slc.step != 1:
-                    raise ValueError("Slicing WCS with a step is not supported.")
+                    raise IndexError("Slicing WCS with a step is not supported.")
```

Comment:
Can we silently (no changelog) change this?

Reply:
I was thinking it hadn't made it to a release, but it was in 3.2, I will add a changelog. Also ping @astrofrog on the change.

Reply:
Looks good